src/config_file.c

```diff
@@ -903,6 +904,26 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 	}
 	g_key_file_remove_key(key_file, "system", "max-bundle-download-size", NULL);
 
+	c->max_bundle_signature_size = g_key_file_get_uint64(key_file, "system", "max-bundle-signature-size", &ierror);
+	if (g_error_matches(ierror, G_KEY_FILE_ERROR, G_KEY_FILE_ERROR_KEY_NOT_FOUND)) {
+		g_debug("No value for key \"max-bundle-signature-size\" in [system] defined "
+				"- using default value of %d bytes.", DEFAULT_MAX_BUNDLE_SIGNATURE_SIZE);
+		c->max_bundle_signature_size = DEFAULT_MAX_BUNDLE_SIGNATURE_SIZE;
+		g_clear_error(&ierror);
+	} else if (ierror) {
+		g_propagate_error(error, ierror);
+		return FALSE;
+	}
+	if (c->max_bundle_signature_size == 0) {
```

Comment:
Checking for "0" explicitly and reporting "Invalid value" with a value parameter looks quite inconsistent. The same applies to having a distinct error code. But I see where this originates from, thus ok for now.